core/src/main/java/hudson/Util.java

```diff
@@ -1168,6 +1169,65 @@ public static FileSet createFileSet(@Nonnull File baseDir, @Nonnull String inclu
         return createFileSet(baseDir,includes,null);
     }
 
+    private static void tryToDeleteSymlink(@Nonnull File symlink) {
+        if (!symlink.delete()) {
+            LogRecord record = new LogRecord(Level.FINE, "Failed to delete temporary symlink {0}");
+            record.setParameters(new Object[]{symlink.getAbsolutePath()});
+            LOGGER.log(record);
```

Comment:
Don't you prefer this way? To avoid logging gets relevance in the code.
`logger.log(Level.FINE, "Failed to delete temporary symlink {0}", new Object[]{new Object[]{symlink.getAbsolutePath()});`

Reply:
I tend to agree.

Reply:
I tend to agree.